humanized_opening_hours/humanized_opening_hours.py

```diff
@@ -331,22 +430,28 @@ def __repr__(self):
 class Moment:
     def __init__(self, moment_type, time_object=None, timedelta=None):
         """
-            A moment in time, which can be fixed or variable, if based
-            on sunrise or sunset. Defined either with a time_object
-            either with a 'sunrise' or 'sunset' type and a timedelta
-            relative to the sunrise or the sunset.
+        A moment in time, which can be fixed or variable, if based
+        on sunrise or sunset. Defined either with a time_object
+        either with a 'sunrise' or 'sunset' type and a timedelta
+        relative to the sunrise or the sunset.
             
-            Parameters
-            ----------
-            moment_type : str
-                The type of the moment, which can be "normal", "sunrise"
-                or "sunset".
-            time_object : datetime.time object, optional
-                The moment itself, required only if 'moment_type'
-                is "normal".
-            timedelta : datetime.timedelta, optional
-                The timedelta relative to the sunrise or the sunset,
-                required only if 'moment_type' is not "normal".
+        Raises
+        ------
+        ValueError
+            moment_type not properly define, time object missing for normal
```

Comment:
`define` => `defined`
`for normal` => `for "normal"` to avoid confusion with the second line